siocli/lib/Siocli.cpp

```diff
@@ -317,15 +360,20 @@ int commandPrompt(const ClientParameters& params)
             // Handle single-word commands
             switch (singleWordCommand) {
                 case SingleWordCommandType::kExit: {
-                    std::cout << "Bye" << std::endl;
+                    std::cout << '\n' << "Bye." << '\n' << std::endl;
```

Comment:
`std::cout << "\nBye." << std::endl;`
I don't think 2 newlines needed here, so drop the one after "Bye"

Reply:
it is nicer to have a blank line before and after "Bye.".  It brings no technical value but gives better relief to the client output. I'll leave it.

Reply:
ok